buddyboss/buddyboss-platform
# PROD-862 Subscription automatically unsubscribes after editing a single Discussion in backend - Fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2102 .

### How to test the changes in this Pull Request:

1. Create a Sample discussion in Forum
2. Click "Subscribe" to the created discussion
3. Go to BuddyBoss > Forums > Discussions and edit the created discussions
4. Go to edited discussion on Frontend and see error is not happening

### Proof Screenshots or Video

https://www.loom.com/share/d318f52900d449c9a4abbc30799c8526

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
